bear.install

```diff
@@ -27,6 +27,9 @@ function bear_install() {
 function _bear_setup_base_configurations() {
   // Set front page to "node".
   \Drupal::configFactory()->getEditable('system.site')->set('page.front', '/node')->save(TRUE);
+  // Install devel submodules.
```

Comment:
It would be better to start a new function parallel to _bear_setup_base_configuration (which is setting variables) like _bear_enable_modules which will enable modules that we want to start out as enabled without setting them to be required.